scripts/configure/zowe-configure-iframe-plugins.sh

```diff
@@ -16,29 +15,24 @@
 
 if [ -z "$NODE_BIN" ]
 then
-  NODE_BIN=${NODE_HOME}/bin/node
+  if [ -n "$NODE_HOME" ]
+  then
+    NODE_BIN=${NODE_HOME}/bin/node
+  else
+    NODE_BIN=node
+  fi
```

Comment:
After configure-node.sh NODE_HOME should always be set, so this shouldn't be needed